src/styles/blocks/card.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  color: black;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 14px;
+  margin-bottom: 48px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__img {
+    width: 160px;
+    padding-bottom: 40px;
```

Comment:
It should be a margin, cause you would like to have outer distance between elements but not inner one. 
so, check you code ALL OVER the document and apply margin between elements instead of padding one